lib/cli/src/Cardano/CLI.hs

```diff
@@ -594,6 +595,27 @@ cmdTransactionList = command "list" $ info (helper <*> cmd) $ mempty
             mTimeRangeEnd
             (ApiT <$> mOrder)
 
+
+-- | Arguments for 'transaction create' command
```

Comment:
Comment should refer to the `transaction submit` command.

Reply:
corrected